### PR TITLE
Add link to GitHub repo

### PR DESCRIPTION
**Pull request by technosophos**
Signed-off-by: Matt Butcher <matt.butcher@fermyon.com>

### CODE REVIEW

File: webassembly-language-support.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ For each environment, we use the following icons to indicate a level of support:
 
 The Fermyon Platform requires [WASI](https://wasi.dev) support. Any language that has a ✅ for WASI should be supported on the Fermyon Platform.
 
+>> If you are interested in contributing to this guide, head on over to [the GitHub repo](https://github.com/fermyon/wasm-languages).
```
Comment (vdice) [inline webassembly-language-support.md:25]: Can/should we also update the link here: https://github.com/fermyon/wasm-languages/blob/main/webassembly-language-support.md#updates-and-additions ?

See https://github.com/fermyon/wasm-languages/issues/11

**Review by vdice — COMMENTED** [review]
Looks good; had one suggestion while we are adding links...

File: webassembly-language-support.md (hunk 1 above)
Comment (technosophos) [reply webassembly-language-support.md:25]: @vdice good idea. I updated this with a more substantial block of text point here to this project, to Discord, and to Twitter.

Closes #11

File: about-examples.md [hunk 2]
```diff
@@ -92,4 +92,6 @@ module = "hello.wasm"
 route = "/"
 ```
 
-Then we start `wagi` with `wagi -c modules.toml`.
\ No newline at end of file
+Then we start `wagi` with `wagi -c modules.toml`.
```
Comment (radu-matei) [inline about-examples.md:95]: TODO (me): update this to use Spin.

**Review by radu-matei — APPROVED** [review]

**Review by vdice — APPROVED** [review]
Other commits crept in it looks like, but fine if there aren't any merge concerns.  LGTM.

File: about-examples.md (hunk 2 above)
Comment (technosophos) [reply about-examples.md:95]: @radu-matei there are both Spin and Wagi instructions here. The Wagi part is just until we finish moving all of our examples over to Spin

**Comment by technosophos** [comment]
@vdice Oops... partly finished rebase. I fixed it and the extra commits are gone

**Pull request merged**